software-delivery-performance-terraform/atlassian.auto.tfvars

```diff
@@ -0,0 +1,62 @@
+# Sumo Logic - SDP Terraform
+# Configure Jira Cloud, Jira Server, OpsGenie, BitBucket credentials and parameters.
+
```

Comment:
Can add a comment: Note: Variables with angle brackets are required.

Reply:
This is tricky, for example, if you do not want to install Jira Server, them Jira server variables are not required.